### PR TITLE
chore(changelog): add changelog for v23 release

### PR DESCRIPTION
**Pull request by joshua-goldstein**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,34 @@ All notable changes to this project will be documented in this file.
...
+## [v23.0.0] - 2023-05-15
+
+### Notes
+
+- This release requires Python >=3.11. Lower versions may also work but are untested
+- The repository has been refactored to use `pyproject.toml` as the source of truth for dependency requirements
+- Added `.python-version` file to keep track of supported Python version
+
+### Added
+- Github Actions for CI/CD pipelines
```
Comment (mangalaman93) [inline CHANGELOG.md:17]: we probably don't need to create the section and one PR is pending that we should merge and add here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,34 @@ All notable changes to this project will be documented in this file.
...
+    - chore(ci): test against latest dgraph on main (#196)
+    - chore(ci): fix test script & use latest docker image (#195)
+    - chore(ci/cd): add pipelines + modernize repo (#193)
+    - chore(ci): add ci to pydgraph (#192)
+- Add example for datetime parsing (#170)
```
Comment (mangalaman93) [inline CHANGELOG.md:22]: same, this could just be indented same as the other lines

File: examples/simple/requirements.txt [hunk 3]
```diff
@@ -1,4 +0,0 @@
-grpcio==1.38.1
```
Comment (mangalaman93) [inline examples/simple/requirements.txt:1]: we should update the README too

File: examples/tls/requirements.txt [hunk 4]
```diff
@@ -1,4 +0,0 @@
-grpcio==1.38.1
```
Comment (mangalaman93) [inline examples/tls/requirements.txt:1]: same here

**Review by mangalaman93 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (skrdgraph) [reply CHANGELOG.md:17]: I think he has a parent bullet for CI/CD pipelines ... and the items below are subbullets to it.

File: pydgraph/meta.py [hunk 5]
```diff
@@ -14,4 +14,4 @@
 
 """Metadata about this package."""
 
-VERSION = '23.0.0rc1'
+VERSION = '23.0.0'
```
Comment (skrdgraph) [inline pydgraph/meta.py:17]: @joshua-goldstein the plan is to have the same branching as our core dgraph project here. 

we should probably add a compatibility matrix in the landing page to show the client <> dgraph version.

Comment (skrdgraph) [reply pydgraph/meta.py:17]: Actually that makes sense ^ .. whatever strategy we follow, it's better to add this to our developer docs for all projects.

Comment (mangalaman93) [reply pydgraph/meta.py:17]: That makes sense. I think the strategy is what you guys decided. I am just saying that clients are too small projects that a linear history has to be good enough. It'd be very difficult for our customers to use the clients if the clients don't follow a linear history. We don't release branches here.

Comment (skrdgraph) [reply pydgraph/meta.py:17]: I agree +1 ^ ...

**Comment by skrdgraph** [comment]
Note: The minimum python version should be >3 ... in pytoml (the current setting will force users to use 3.11+ today, which is not ideal)

File: CHANGELOG.md (hunk 1 above)
Comment (joshua-goldstein) [reply CHANGELOG.md:17]: @mangalaman93 they are all related to the same solution (adding ci/cd pipelines).  It is more clear this way

**Comment by joshua-goldstein** [comment]
> Note: The minimum python version should be >3 ... in pytoml (the current setting will force users to use 3.11+ today, which is not ideal)

I am setting the minimum to 3.7 now, since this is the minimum required by the versions of grpcio and protobufs that we are pinned to.

File: examples/tls/requirements.txt (hunk 4 above)
Comment (joshua-goldstein) [reply examples/tls/requirements.txt:1]: this file has been removed and readme updated

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,8 +5,40 @@ All notable changes to this project will be documented in this file.
...
+## [v23.0.0] - 2023-05-15
+
+### Notes
+
+- Testing is done against Python >=3.11. Minimum requirement is now Python 3.5
+- The repository has been refactored to use `pyproject.toml` as the source of truth for dependency requirements
+- Added `.python-version` file to keep track of supported Python version
+
+### Breaking
+- accept grpc endpoint or /graphql endpoint in from_cloud function (#190)
```
Comment (mangalaman93) [inline CHANGELOG.md:17]: We also remove  the slash function, we should mention that too. Looks good otherwise.

**Review by mangalaman93 — DISMISSED** [review]

**Review by mangalaman93 — APPROVED** [review]

**Review by mangalaman93 — APPROVED** [review]

**Pull request merged**